ui/src/api/SdkRequests.ts

```diff
@@ -3,6 +3,10 @@ import expressApi from 'axios'
 
```

Comment:
If the device is offline, the env var checks will probably report a bunch of unfriendly errors. There is the internet check variable that can be checked to see if the device is online, and if not return a friendlier message. 

```
import { internetConnectivity } from '../api/SystemRequests'

if (internetConnectivity.status) {
  console.log('you have internet')
} else if (internetConnectivity.status === false) {
  console.log('you DO NOT have internet')
} else if (internetConnectivity.status === undefined) {
  console.log(
    "The very first internet check at launch hasn't completed yet."
  )
}
```

I wonder if there is a way to apply this to all the SDK endpoints, rather than having to write the check in for each request?

https://github.com/maggie0002/balena-device-ui/wiki/Internet-connectivity-check